diff -urN a/net/netfilter/nf_conntrack_core.c b/net/netfilter/nf_conntrack_core.c
--- a/net/netfilter/nf_conntrack_core.c	2022-01-21 11:42:50.153260211 +0800
+++ b/net/netfilter/nf_conntrack_core.c	2022-01-21 11:44:10.901703527 +0800
@@ -1487,6 +1487,7 @@
 	/* save hash for reusing when confirming */
 	*(unsigned long *)(&ct->tuplehash[IP_CT_DIR_REPLY].hnnode.pprev) = hash;
 	ct->status = 0;
+	memset(&ct->fros_data, 0x0, sizeof(struct nf_fros_data));
 	WRITE_ONCE(ct->timeout, 0);
 	write_pnet(&ct->ct_net, net);
 	memset(&ct->__nfct_init_offset, 0,
